src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/wwwroot/pdf_viewer.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  <title>PDF viewer</title>
+
+  <style>
+    body {
+      background-color: #808080;
+      margin: 0;
+      padding: 0;
+    }
+    #viewerContainer {
+      overflow: auto;
+      position: absolute;
+      width: 100%;
+      height: 100%;
+    }
+  </style>
+
+  <link rel="stylesheet" href="./pdfjs-dist/dist/web/pdf_viewer.css">
+
+  <script src="./pdfjs-dist/dist/build/pdf.mjs" type="module"></script>
+  <script src="./pdfjs-dist/dist/web/pdf_viewer.mjs" type="module"></script>
+</head>
+<body tabindex="1">
+  <div id="viewerContainer">
+    <div id="viewer" class="pdfViewer"></div>
+  </div>
+
+  <script src="pdf_viewer.mjs" type="module"></script>
```

Comment:
There are two JS files both called `pdf_viewer.mjs`? (This one and the one on L25)

Assuming these aren't duplicates, would it work to give them different names? It could get confusing, e.g., when working on this or if people are trying to upgrade or report issues otherwise.

Reply:
Good catch, I'll change their names.